catalog/plugin/catalog-plugin-query-monitor-commands/src/test/java/ddf.catalog.querymonitorcommands/QueryMonitorCommandsTest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.querymonitorcommands;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import java.util.UUID;
+
+import org.junit.Test;
+
+import junit.framework.Assert;
+
+import ddf.catalog.querymonitor.ActiveSearch;
+import ddf.catalog.querymonitor.QueryMonitorPluginImpl;
+
+public class QueryMonitorCommandsTest {
+
+    QueryMonitorPluginImpl qmpi = new QueryMonitorPluginImpl();
+
+    ActiveSearch as = null;
+
+    ByteArrayOutputStream baos;
+
+    @Test
+    public void testPrintSearchTableToConsole() {
+        QueryMonitorCommandPrintTable printCommand = new QueryMonitorCommandPrintTable(qmpi) {
+            @Override
+            PrintStream getPrintStream() {
+                baos = new ByteArrayOutputStream();
+                return new PrintStream(baos);
+            }
+        };
+        qmpi.setRemoveFromTable(false);
+        UUID u = UUID.randomUUID();
+        as = new ActiveSearch("someQuery", null, u);
+        qmpi.addActiveSearch(as);
+        printCommand.printSearchTableToConsole();
+        Assert.assertTrue(baos.toString()
```

Comment:
We should use `assertThat()` instead of the other various `assert` methods since it's more expressive and it gives more detailed error messages on test failures.